### PR TITLE
Make the package UPM compatible

### PR DESCRIPTION
**Pull request by zeh**
This PR...

* Makes unity-wakatime conform to UPM package's format, so it will be able to be installed as a package in the future. Right now it can be installed by copying the files locally, but in the future uses should be able to [install directly from git](https://forum.unity.com/threads/git-support.530344/) by simply adding a URL to their `manifest.json` files.
* Moves files around to a different folder, just so it works for both 2018.1+ and other versions of Unity.
* Add `#if (UNITY_EDITOR)`...`#endif` directives so the plugin's code is not exported or ran with built games (this was causing problems and should not be necessary).

More discussion on the package manager can be found on issue #5.

Let me know what you think.

### CODE REVIEW

File: com.vladfaust.unitywakatime/com.vladfaust.unitywakatime.asmdef [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿{
+	"name": "NewAssembly"
```
Comment (vladfaust) [inline com.vladfaust.unitywakatime/com.vladfaust.unitywakatime.asmdef]: Any comments on why using `NewAssembly`? Maybe leaving blank? I'm not strong in C#...

File: com.vladfaust.unitywakatime/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "unity": "2018.1",
+    "description": "WakaTime plugin for Unity",
+    "keywords": [
+        "unity",
+		"wakatime",
+		"stats",
+		"tracker",
+		"programming"
+    ],
+    "category": "Unity"
```
Comment (vladfaust) [inline com.vladfaust.unitywakatime/package.json]: Is that a best category name? Maybe something like "Productivity"? :thinking:

Comment (zeh) [reply com.vladfaust.unitywakatime/package.json]: That's a good point too. I'm not sure there's "sanctioned" category names yet, but "Productivity" makes sense.

File: com.vladfaust.unitywakatime/com.vladfaust.unitywakatime.asmdef (hunk 1 above)
Comment (zeh) [reply com.vladfaust.unitywakatime/com.vladfaust.unitywakatime.asmdef]: Good question. This is just the internal name that Unity has given to the thing. Not sure what the contents of this file are. I'll investigate a bit more.

Comment (vladfaust) [reply com.vladfaust.unitywakatime/com.vladfaust.unitywakatime.asmdef]: Try to import into a project without this file

Comment (zeh) [reply com.vladfaust.unitywakatime/com.vladfaust.unitywakatime.asmdef]: Technically we probably can make do without the file, but it's [recommended by most people](https://gist.github.com/LotteMakesStuff/6e02e0ea303030517a071a1c81eb016e#file-5-md) and a good idea to have them there as using assembly definition files should [increase build performance](https://blogs.unity3d.com/2017/11/22/unity-2017-3b-feature-preview-assembly-definition-files-and-transform-tool/).

I've given it a unique name.

**Comment by zeh** [comment]
@vladfaust All updated, was just getting some other stuff done first.

You might need to [add a `CONTRIBUTING` file to your repository](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) to make it more clear on how to contribute, state that you're following AngularJS's commit conventions, etc.

**Comment by vladfaust** [comment]
Nah, nevermind. Thanks for your contribution, @zeh!

**Comment by zeh** [comment]
Thanks @vladfaust!

**Pull request merged**